### PR TITLE
Linters Check

### PR DESCRIPTION
**Pull request by Phildeoner**
Testing lInters checkers

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```
Comment (DanSam5K) [inline .github/workflows/linters.yml]: - Delete this line and try again.

**Review by DanSam5K — COMMENTED** [review]
## Hi @Phildeoner,
- Good Job check the comment and try lets see if it works!

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Phildeoner) [reply .github/workflows/linters.yml]: Yes sir changes effected and it works, thank you.

**Comment by Phildeoner** [comment]
Thank you sir, Linter works perfectly and passed all checks.

**Pull request merged**